mac

```diff
@@ -86,7 +86,6 @@ gem_install_or_update() {
     gem update "$@"
   else
     gem install "$@"
-    rbenv rehash
```

Comment:
I haven't needed it myself either, but I understand it to be the equivalent in asdf. I'm fine with it being removed, but I was wondering if there was a reason to remove rather than replace.